pkg/base/cockpit.js

```diff
@@ -2274,6 +2274,25 @@ function full_scope(cockpit, $, po) {
         /* 1024: [ null, "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB" ] */
     };
 
+    cockpit.bytes_from_format = function bytes_from_format(formatted, factor, separate) {
```

Comment:
Isn't this docker specific? Can we leave it in docker/docker.js for now? Or do you have plans to use this somewhere else? In general "screen scraping" is unfortunate and usually a work around for a junky API. Ideally we won't be using this code in many places? Or am I wrong?